docs/self-hosted-appcircle/self-hosted-runner/runner-vm-setup.md

```diff
@@ -512,6 +512,83 @@ tart clone macOS_240514 vm01
 
 In docker terminology, `vm01` and `vm02` will be our docker images. We will configure them separately, persist our changes and then create containers to execute build pipelines. On every build, fresh containers will be used for both runners.
 
+### Configure Runner VM Resources
+
+You can adjust the resource limits for Runner VMs based on your needs.
+
+:::info 
+By default, our runner images are configured with an 8GB memory limit and 4 CPU cores.
+:::
+
+:::caution 
+Total allocated resources (memory, cpu) for all VMs combined, should not exceed host machine's physical capacity for optimal performance.
+:::
+
+#### Set Memory Limits
+
+To configure the memory limit for a VM, run the following command:
+
+   ```bash
+   tart set <vm_name> --memory <size_in_mb>
+   ```
+
+   Replace `vm_name` with your VM's name and `size_in_mb` with the desired memory size in MB. E.g., 
+   > `tart set vm01 --memory 8192` (8GB)
+
+   > `tart set vm01 --memory 16384` (16GB)
+
+:::tip
+Example configurations:
+
+| Host Memory Size | Runner Memory Config. |
+|------------------|-----------------------|
+| 8GB              | 1 x VM with 8GB       |
+| 16GB             | 2 x VM with 8GB       |
+| 16GB             | 1 x VM with 16GB      |
+| 32GB             | 2 x VM with 16GB      |
+:::
+
+#### Set CPU Limits
+
+To configure the number of CPU cores for a VM, run the following command:
+
+   ```bash
+   tart set <vm_name> --cpu <count>
+   ```
+
+   Replace `vm_name` with your VM's name and `count` with the desired number of CPU cores. E.g., 
+   > `tart set vm01 --cpu 4`
+
+   > `tart set vm01 --cpu 8`
+
+:::tip
+To check the number of CPU cores on your system, use the following command:
+```bash
+sysctl -n hw.ncpu
+```
+
+Example configurations:
```

Comment:
I think we should describe this example configurations before the tart set command since the users should see the example before they set limit. 

Also we should describe how they can use output of the `sysctl -n hw.ncpu` command. For example: "If you have 10 core CPU according to the output and you plan to run 2 VMs, divide the CPU count to the half for CPU count per VM. If you plan to use one VM only, you can use total CPU limit for the VM. "

Reply:
Added examples in text for further clarification.